pkg/web_app/lib/src/search.dart

```diff
@@ -5,12 +5,28 @@
 import 'dart:html';
 
 void setupSearch() {
+  _setEventForKeyboardShortcut();
   _setEventForSearchInput();
   _setEventForFiltersToggle();
   _setEventForSortControl();
   _setEventForCheckboxChanges();
 }
 
+void _setEventForKeyboardShortcut() {
+  final inputElem = document.querySelector('input.site-header-search-input');
+  if (inputElem != null && inputElem is InputElement) {
+    document.onKeyPress.listen((e) {
```

Comment:
```suggestion
    window.onKeyPress.listen((e) {
```

Reply:
Might be worth testing that this doesn't prevent us from writing `/` in other input fields on the page.

I think it will.. We probably have to test what target the event is hitting, otherwise you'll steal form field focus from any field the user is trying to write slash in.

Reply:
Good catch! I've updated with checking the active element differently. PTAL.